controller/workitems.go

```diff
@@ -123,7 +123,9 @@ func (c *WorkitemsController) Create(ctx *app.CreateWorkitemsContext) error {
 			return errs.Wrap(err, fmt.Sprintf("Error creating work item"))
 		}
 
-		wi, err = appl.WorkItems().Create(ctx, ctx.SpaceID, *wit, wi.Fields, *currentUserIdentityID)
+		// if the revision ID has to be sent to the notification service
+		// then pass the same value to the `notification.NewWorkItem*`.
+		wi, err = appl.WorkItems().Create(ctx, ctx.SpaceID, *wit, wi.Fields, *currentUserIdentityID, uuid.NewV4())
```

Comment:
For a new `WorkitemCreated` we don't need to send revision id because just using the workitem id we could figure out what is created.

Revision is needed only on update, since things have changed in the workitem and what has changed is figured out using revision id.